Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM yastdevel/cpp
+RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
+  "pkgconfig(gio-unix-2.0)" \
+  "pkgconfig(glib-2.0) >= 2.31.0" \
+  "pkgconfig(gtk+-3.0)" \
+  "pkgconfig(libgnome-menu-3.0)"
```

Comment:
Moreover you would need to handle conditions like
```
%if 0%{?suse_version} < 1220
BuildRequires:  libxcrypt-devel
%endif
```

(taken from [here](https://github.com/yast/yast-slp/blob/master/package/yast2-slp.spec#L31)).